documentation/smg/16_serialiser.md

```diff
@@ -0,0 +1,229 @@
+# The serialiser
+
+In order to efficiently transfer MATLAB information between storage, be it via networked interaction (such as MPI) or to disc efficiently, the information is first "serialised" into a representation. This document details the operation of the serialisers defined in:
+```
+<HERBERT_ROOT>/herbert_core/utilities/misc/hlp_serialise
+<HERBERT_ROOT>/herbert_core/utilities/misc/hlp_deserialise
+<HERBERT_ROOT>/_LowLevelCode/cpp/cpp_communicator/serialiser/c_serialise
+<HERBERT_ROOT>/_LowLevelCode/cpp/cpp_communicator/serialiser/c_deserialise
+```
+
```

Comment:
Thanks for this. Looking at the tickets, I was expecting you to clarify that the serialization is for MPI data transfer, and it is very useful to know that it is intended to be generic.